test/unit/kubernetes-deploy/google_friendly_config_test.rb

```diff
@@ -14,7 +14,7 @@ def teardown
   def test_auth_use_default_gcp_success
     config = KubernetesDeploy::KubeclientBuilder::GoogleFriendlyConfig.new(kubeconfig, "")
 
-    stub_request(:post, 'https://www.googleapis.com/oauth2/v4/token')
+    stub_request(:post, 'https://oauth2.googleapis.com/token')
```

Comment:
This is a legitimate change introduced by https://github.com/googleapis/google-auth-library-ruby/pull/147.

Note that there isn't much in the way of docs in that PR, and the old version we used (0.6.2) has no tag associated with it.